### ISSUE TITLE
Highlight single-line docblocks

### ISSUE THREAD

**Issue by daltones** [atom/language-php#135]
I think it's common write `@var` tags in a single-line docblock for class properties. PHP Documentor actually suggests an example here: https://www.phpdoc.org/docs/latest/references/phpdoc/tags/var.html.

I don't know why TextMate based syntaxes have this behavior, but maybe atom should accept single-line docblocks at all and highlight all `@` tags.

``` php
/**
 * @var type Description
 */
protected $property;
```

``` php
/** @var type Description */        // No highlight here!
protected $property;
```

**Comment by kafoso** [comment]
Agreed. Would very much appreciate this.

May also be useful for adding references to ordinary variables in single-line comments, e.g. in loops as this one:

```php
foreach ($users as $user) { // @var $user My\Application\Entity\User
    // ...
}
```

`@override` also comes to mind, which isn't highlighted, either.

I'm aware I'm using a single-line comment in the above example. However, I am very strongly against using multiline comments when a single line will suffice. Simply because of debugging annoyances. Below code will break down when wrapped in a multi-line comment while the `@var` reference is wrapped in a multi-line comment, too:

```php
/*
foreach ($users as $user) { /** @var $user My\Application\Entity\User */
    // ...
}
*/
```

Of course, restrictions should be enforced so that single-line comments won't get sporadially colored. E.g.:

1. Start with two slashes (not inside a string).
2. One space character (and not tab).
3. Notation, e.g. `@var` or `@override`.
4. One space character (and not tab).
5. The remainder of the comment, formatted as usual.

**Comment by 50Wliu** [comment]
Fixed by #191.
